resources/recipes.py

```diff
@@ -113,13 +113,16 @@ def item(thing: str):
         def plank(thing: str):
             return 'tfc:wood/planks/%s_%s' % (wood, thing)
 
-        rm.crafting_shaped('crafting/wood/%s_bookshelf' % wood, ['XXX', 'YYY', 'XXX'], {'X': item('planks'), 'Y': 'minecraft:book'}, plank('bookshelf')).with_advancement('minecraft:book')
+        def log_tag():
+            return '#tfc:%s_logs' % wood
```

Comment:
```suggestion
        log_tag = '#tfc:%s_logs' % wood
```
There's no reason for this to be a inline `def`